src/mono/browser/runtime/diagnostics/index.ts

```diff
@@ -30,8 +30,13 @@ export function setRuntimeGlobals (globalObjects: GlobalObjects): void {
         return 0;
     };
 
-    diagnosticHelpers. ds_rt_websocket_close = (client_socket :number):number => {
+    diagnosticHelpers.ds_rt_websocket_close = (client_socket :number):number => {
         // Not implemented yet
         return 0;
     };
+    runtimeHelpers.mono_wasm_instrument_method = (method: MonoMethod): number => {
```

Comment:
This function runs very often, right?
Shouldn't it use some variable computed on start instead of always "computing" the value?

Reply:
This runs once for each method, so it's not too bad. This is just temporary version. On next PR there is more complex version which could do filtering on it via regex, and that is cached.